.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+Tag name (required for release branches):
+Originator(s):
+
+Summary (include the keyword ['closes', 'fixes', 'resolves'] and issue number):
```

Comment:
I believe we should include the issue title as well (a pain to copy/type, but very important for getting a summary of what issue xyz is all about)

Reply:
done!